activesupport/lib/active_support/callbacks.rb

```diff
@@ -66,6 +67,12 @@ module Callbacks
 
     CALLBACK_FILTER_TYPES = [:before, :after, :around]
 
+    # If +true+, Active Record and Active Model callbacks returning +false+ will
```

Comment:
We reserve `+true+` for values that are `== true` and not just truthy. You should revert the places you added `+`s back to where they were before :wink: